GVFS/GVFS.Common/NuGetUpgrade/NuGetUpgrader.cs

```diff
@@ -124,25 +129,9 @@ public class NuGetUpgrader : ProductUpgrader
             {
                 error = $"NuGetUpgrader: Unable to locate git installation. Ensure git is installed and try again.";
                 return false;
-            }
-
-            string authUrl;
-            if (!TryCreateAzDevOrgUrlFromPackageFeedUrl(upgraderConfig.FeedUrl, out authUrl, out error))
-            {
-                return false;
-            }
-
-            if (!TryGetPersonalAccessToken(
-                    gitBinPath,
-                    authUrl,
-                    tracer,
-                    out string token,
-                    out string getPatError))
-            {
-                error = $"NuGetUpgrader was not able to acquire Personal Access Token to access NuGet feed. Error: {getPatError}";
-                tracer.RelatedError(error);
-                return false;
-            }
+            }
+
+            CredentialManager credentialManager = new CredentialManager(new GitProcess(gitBinPath, null, null));
```

Comment:
This might fall under one of your TODOs, but it looks like this method is eventually hit in the `UpgradeOrchastratorTests`, and so you might want to make `credentialManager` a parameter so that the unit tests can pass in a mock `CredentialManager` (or a real `CredentialManager` created with a mock `GitProcess`).

1.  Unit test calls`this.orchestrator.Execute()`
2. `Execute` calls `UpgradeOrchestrator.TryInitialize()`
3. `TryInitialize` calls `ProductUpgrader.TryCreateUpgrader`
4. `TryCreateUpgrader` calls this method (`NuGetUpgrader.TryCreate`)